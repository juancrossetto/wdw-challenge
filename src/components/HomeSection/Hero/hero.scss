@use '../../../scss/' as *;

.hero {
  background-color: $primary;
  height: 208px;
  width: 100%;
  position: relative;
  pointer-events: none;
  background: url('../../../assets/images/hero-bg.jpg');
  background-repeat: repeat;
  background-size: cover;

  @include for-phone-only {
    height: 120px;
  }
  &__content {
    position: absolute;
    bottom: 10px;
    padding-left: 32px;

    @include for-phone-only {
      bottom: -30px;
    }
    &__text {
      color: $white;
      width: 199px;
      height: 73px;
      font-style: normal;
      font-weight: bold;
      font-size: 60px;
      line-height: 70px;
      letter-spacing: 0.01em;
      @include for-phone-only {
        font-style: normal;
        font-weight: 700;
        font-size: 35px;
        line-height: 41px;
        letter-spacing: 0.01em;
      }
    }
  }
}
