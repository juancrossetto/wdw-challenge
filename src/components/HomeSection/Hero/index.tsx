import React from 'react';
import './hero.scss';

const Hero = () => {
  return (
    <div className="hero">
      <div className="container p-0">
        <div className="hero__content">
          <h1 className={'hero__content__text'}>Gallery</h1>
        </div>
      </div>
    </div>
  );
};

export default Hero;
