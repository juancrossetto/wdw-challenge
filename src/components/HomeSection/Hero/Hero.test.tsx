import React from 'react';
import renderer from 'react-test-renderer';

import Hero from '.';

describe('Hero component', () => {
  test('Hero render correctly', () => {
    const tree = renderer.create(<Hero />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
