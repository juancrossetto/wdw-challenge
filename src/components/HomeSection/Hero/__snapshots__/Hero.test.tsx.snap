// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component Hero render correctly 1`] = `
<div
  className="hero"
>
  <div
    className="container p-0"
  >
    <div
      className="hero__content"
    >
      <h1
        className="hero__content__text"
      >
        Gallery
      </h1>
    </div>
  </div>
</div>
`;
